swagger.yaml

```diff
@@ -3622,6 +3622,7 @@ definitions:
               - 'DISK_ENCRYPTION'
               - 'CLUSTER_MANAGED_NETWORKING_WITH_VMS'
               - 'ARM64_ARCHITECTURE'
+              - 'ARM64_ARCHITECTURE_WITH_CLUSTER_MANAGED_NETWORKING'
```

Comment:
can we move all this enum to a different place in the swagger, such as we have in  host-validation-id? I think it will change the name from FeatureSupportLevelFeaturesItems0FeatureIDCLUSTERMANAGEDNETWORKINGWITHVMS to something more readable FeatureSupportIdCLUSTERMANAGEDNETWORKINGWITHVMS

Reply:
we can but it will require api change